mw-webapp/src/component/notification/Notification.tsx

```diff
@@ -0,0 +1,69 @@
+import {ReactElement} from "react";
+import {Cross2Icon} from "@radix-ui/react-icons";
+import {Close, Description, Provider, Root, Viewport} from "@radix-ui/react-toast";
+import styles from "src/component/notification/Notification.module.scss";
+
+/**
+ * Control the sensitivity of the toast for accessibility purposes.
+ * Foreground - for toasts that are the result of a user action.
+ * Background - for toasts generated from background tasks .
+ */
+export enum NotificationType {
+  "foreground" = "foreground",
+  "background" = "background",
+}
+
+/**
+ * Notification props
+ */
+interface NotificationProps {
+
+  /**
+   * The time in milliseconds that should elapse before automatically closing each toast.
+   * @default 5000
+   */
+  duration?: number;
+
+  /**
+   * An author-localized label for each toast.
+   * Used to help screen reader users associate the interruption with a toast.
+   * @default "Notification"
+   */
+  label?: string;
+
+  /**
+   * The content to display within the notification.
+   */
+  description: ReactElement<HTMLElement>;
+
+  /**
+   * Notification type
+   * @default "foreground"
+   */
+  type?: NotificationType;
+}
+
+/**
+ * Notification component
+ */
+export const Notification = (props: NotificationProps) => {
+  return (
+    <Provider
+      duration={props.duration}
+      label={props.label}
+    >
+      <Root className={styles.root}>
+        <Description asChild>
+          {props.description}
```

Comment:
What if you will want to show not only one string in notification? Notifications could include title, different styled text even images. How you will get it from just one prop description?

Reply:
description(already content) is ReactElement<HTMLElement>. all of the above can be in it))